<curl/types.h> went away in curl 7.22.0; everything in it should be
available through <curl/curl.h> anyway.

--- GTorrentViewer-0.2b/src/main.c
+++ GTorrentViewer-0.2b/src/main.c
@@ -40,7 +40,6 @@
 #include <errno.h>
 
 #include <curl/curl.h>
-#include <curl/types.h>
 #include <curl/easy.h> 
 
 #include "bencode.h"
